### PR TITLE
1115 proofreading changes 3

### PR DESCRIPTION
**Pull request by Robinlovelace**
- **Be more specific and accurate: raster and vector datasets *used* in many fields**
- ** as it -> , as it**
- **3-d -> three-d***
- **-0.1 -> $-0.1$**
- **(the Prime M...) -> (Prime M...)**
- **peninsular -> peninsula**
- **low level libraries -> low-level libraries**
- **Fix-up wording about importance of GDAL and friends**
- **Tweaks**
- **2-d -> two-d**
- **Add missing comma**
- **in Sec -> Sec**
- **Remove hyperlink from the**
- **Time of writing,**
- **Update at the time of writing**
- **re-iterate -> reiterate**
- **CRSs abbreviation**
- **A spatial -> Spatial**
- **Shorten description of sticky**
- **CRSs**
- **this reason -> this reason,**
- **Neither has**
- **Fix grammar, mention border arg of plot()**
- **Basic maps**
- **A plot -> Plot**
- **Base plot limits**
- **Shift index entries**
- **7 -> seven**
- **two-, three-, or four-dimensional space**
- **From scratch,**
- **Tweaks to 2.2.5**
- **Update hyperlink**
- **Examples, the**
- **Assorted typo fixes**
- **Assorted changes**
- **multilayered -> multi-layered and more**
- **the Zion -> Zion**
- **Mention fig:basic-new-raster-plot in the text**
- **multilayer -> multi-layer**
- **Tweaks to 2.4**
- **Spell-out UTM, finish c2**

### CODE REVIEW

File: 02-spatial-data.Rmd [hunk 1]
```diff
@@ -95,63 +95,64 @@ This book uses **sf** and **terra** packages to work with vector data and raster
...
 
 ```{r vectorplots-source, include=FALSE, eval=FALSE}
 source("https://github.com/geocompx/geocompr/raw/main/code/02-vectorplots.R") # generate subsequent figure
 ```
 
-```{r vectorplots, fig.cap="Illustration of vector (point) data in which the location of London (the red X) is represented with reference to an origin (the blue circle). The left plot represents a geographic CRS with an origin at 0° longitude and latitude. The right plot represents a projected CRS with an origin located in the sea west of the South West Peninsula.", out.width="49%", fig.show='hold', echo=FALSE, fig.scap="Illustration of vector (point) data."}
+```{r vectorplots, fig.cap="V ector (point) data in which the location of London (red X) is represented with reference to an origin (blue circle). The left plot represents a geographic CRS with an origin at 0° longitude and latitude. The right plot represents a projected CRS with an origin located in the sea west of the South West Peninsula.", out.width="49%", fig.show='hold', echo=FALSE, fig.scap="Vector (point) data."}
```
Comment (Nowosad) [inline 02-spatial-data.Rmd:126]:
```suggestion
```{r vectorplots, fig.cap="Vector (point) data in which the location of London (red X) is represented with reference to an origin (blue circle). The left plot represents a geographic CRS with an origin at 0° longitude and latitude. The right plot represents a projected CRS with an origin located in the sea west of the South West Peninsula.", out.width="49%", fig.show='hold', echo=FALSE, fig.scap="Vector (point) data."}
```

File: 02-spatial-data.Rmd [hunk 2]
```diff
@@ -204,7 +205,7 @@ The contents of this `geom` column give `sf` objects their spatial powers: `worl
 Although part of R's default installation (base R), `plot()` is a [*generic*](https://adv-r.hadley.nz/s3.html#s3-methods) that is extended by other packages.
 **sf** contains the non-exported (hidden from users most of the time) `plot.sf()` function which is what is called behind the scenes in the following command, which creates Figure \@ref(fig:world-all).
 
-```{r world-all, fig.cap="A spatial plot of the world using the sf package, with a facet for each attribute.", warning=FALSE, fig.scap="A spatial plot of the world using the sf package."}
+```{r world-all, fig.cap="Spatialplot of the world using the sf package, with a facet for each attribute.", warning=FALSE, fig.scap="Spatialplot of the world using the sf package."}
```
Comment (Nowosad) [inline 02-spatial-data.Rmd:208]: I am a bit surprised by this..

File: 02-spatial-data.Rmd [hunk 3]
```diff
@@ -1019,7 +1020,7 @@ multi_rast
 nlyr(multi_rast)
 ```
 
-For multilayer raster objects, layers can be selected with the `[[` and `$` operators, for example with commands `multi_rast[["landsat_1"]]` and `multi_rast$landsat_1`.
+For multi-layerraster objects, layers can be selected with the `[[` and `$` operators, for example with commands `multi_rast[["landsat_1"]]` and `multi_rast$landsat_1`.
```
Comment (Nowosad) [inline 02-spatial-data.Rmd:1023]: Again, I am a bit surprised by this

File: 02-spatial-data.Rmd [hunk 4]
```diff
@@ -1111,11 +1112,11 @@ It is typically used in mapping polar regions.
 
 A quick summary of different projections, their types, properties, and suitability can be found at [www.geo-projections.com](https://www.geo-projections.com/).
 We will expand on CRSs and explain how to project from one CRS to another in Chapter \@ref(reproj-geo-data).
-For now, it is sufficient to know:
+For now, it is sufficient to know that:
```
Comment (Nowosad) [inline 02-spatial-data.Rmd:1115]: I think the list still can be improved

File: 03-attribute-operations.Rmd [hunk 5]
```diff
@@ -405,13 +405,13 @@ These aggregating functions return `sf` objects with rows representing continent
...
 Let's combine what we have learned so far about **dplyr** functions, by chaining multiple commands to summarize attribute data about countries worldwide by continent.
-The following command calculates population density (with `mutate()`), arranges continents by the number countries they contain (with `arrange()`), and keeps only the 3 most populous continents (with `slice_max()`), the result of which is presented in Table \@ref(tab:continents)):
+The following command calculates population density (with `mutate()`), arranges continents by the number of countries they contain (with `arrange()`), and keeps only the three most populous continents (with `slice_max()`), the result of which is presented in Table \@ref(tab:continents)):
 
 ```{r 03-attribute-operations-30}
 world_agg5 = world |> 
   st_drop_geometry() |>                      # drop the geometry for speed
   select(pop, continent, area_km2) |> # subset the columns of interest  
-  group_by(continent) |>                     # group by continent and summarize:
+  group_by(Continent = continent) |>  # group by continent and summarize:
```
Comment (Nowosad) [inline 03-attribute-operations.Rmd:414]: @Robinlovelace would you write your own code like this?

File: 02-spatial-data.Rmd (hunk 2 above)
Comment (Robinlovelace) [reply 02-spatial-data.Rmd:208]: Yeah my bad good spot will fix

File: 03-attribute-operations.Rmd (hunk 5 above)
Comment (Robinlovelace) [reply 03-attribute-operations.Rmd:414]: Yes, if the aim was to have "Continent" consistently capitalised in the resulting table. How would you write it @Nowosad?

File: 02-spatial-data.Rmd (hunk 4 above)
Comment (Robinlovelace) [reply 02-spatial-data.Rmd:1115]: Probably.

**Comment by Robinlovelace** [comment]
Going to merge to keep the momentum. Post-merge review welcome.

**Comment by Nowosad** [comment]
Hi @Robinlovelace -- I think this PR introduces several new issues, for example:

- "multi-layerobject"
- "multi-layerfiles"
- "multi-layerfile"
- "multi-layer`SpatRaster`"
- "multi-layerSpatRaster"

I do not know if I found all of them -- could you double check and correct it?

**Comment by Robinlovelace** [comment]
@Nowosad see https://github.com/geocompx/geocompr/pull/1132/commits/31807ba30dc3a05eace3fdca155fe52a684fb762

**Comment by Nowosad** [comment]
👍🏻

**Pull request merged**